### PR TITLE
Don't inform admins about MB cloud on EE instances

### PR DESCRIPTION
**Pull request by mazameli**
This just adds checks to not show this info when it's not pertinent.

Marked as draft 'cause I'm not sure how to switch my local instance to EE to check that this works.

### CODE REVIEW

**Comment by paulrosenzweig** [comment]
The `versionInfo` wasn't working for me locally. I added a more foolproof check with a new `isEnterprise` setting. It's not actually a setting but I put it in there since it related. It essentially just checks what version of the FE was built.

File: frontend/test/__support__/cypress.js [hunk 1]
```diff
@@ -222,6 +222,10 @@ export const describeWithToken = Cypress.env("HAS_ENTERPRISE_TOKEN")
   ? describe
   : describe.skip;
 
+export const itOpenSourceOnly = Cypress.env("HAS_ENTERPRISE_TOKEN")
+  ? it.skip
+  : it;
+
```
Comment (paulrosenzweig) [inline frontend/test/__support__/cypress.js:228]: @nemanjaglumac thoughts on this? It's the opposite of `describeWithToken` but for `it`. I figured that EE-behavior is typically tested in multiple test blocks, weird OSS-only behavior like this is probably a single test. I still think the it-vs-describe difference is a bit weird though.

Comment (nemanjaglumac) [reply frontend/test/__support__/cypress.js:228]: I am ok with it. It's just the name that feels so weird. I'm expecting that it "opens" the source of the code that follows :D
Having said that, I can't think of a nice name.

**Review by kdoh — APPROVED** [review]

File: frontend/test/__support__/cypress.js (hunk 1 above)
Comment (paulrosenzweig) [reply frontend/test/__support__/cypress.js:228]: Yeah agreed the name is weird. I don't expect it to be commonly used, but maybe a better name will come to us whenever it's next needed.

**Comment by kdoh** [comment]
Got a 👍 from Sameer on slack.

**Pull request merged**